admin/server/app/createStaticRouter.js

```diff
@@ -84,13 +87,12 @@ module.exports = function createStaticRouter (keystone) {
 
 	/* Configure router */
 	router.use('/styles', less(path.resolve(__dirname + '/../../public/styles'), lessOptions));
-	router.use('/styles/fonts', express.static(path.resolve(__dirname + '/../../public/js/lib/tinymce/skins/keystone/fonts')));
```

Comment:
This path [doesn't exist any more](https://github.com/keystonejs/keystone/tree/master/admin/public/js/lib/tinymce/skins/keystone/fonts) so I pointed it to the `keystone-tinymce` fonts